libfwupd/fwupd-client.c

```diff
@@ -4471,12 +4517,12 @@ fwupd_client_class_init (FwupdClientClass *klass)
 	/**
 	 * FwupdClient:soup-session:
 	 *
-	 * The libsoup session.
+	 * The libsoup session, now unused.
 	 *
 	 * Since: 1.4.5
 	 */
-	pspec = g_param_spec_object ("soup-session", NULL, NULL, SOUP_TYPE_SESSION,
-				     G_PARAM_READWRITE | G_PARAM_STATIC_NAME);
+	pspec = g_param_spec_object ("soup-session", NULL, NULL, G_TYPE_OBJECT,
+				     G_PARAM_READABLE | G_PARAM_STATIC_NAME);
```

Comment:
isn't this also ABI breaking?  I guess _gnome-software didn't actually use it though, so don't make too much noise_?

Reply:
> so don't make too much noise

Right! Sssh! :)